### DISCUSSION TITLE
Increase intellisense window

### DISCUSSION THREAD

**Discussion by kjoebie** [DaxStudio/DaxStudio#950] [Ideas]
Hi All,

Is it possible to manually increase the intellisense window? I'm using rather long measure names and they are not very well shown in the intellisense window:
![image](https://user-images.githubusercontent.com/7207207/191426439-590a5113-26bd-4426-8f70-1896781a252b.png)

There is a scrollbar showing, but then the measures become even more difficult to read or choose from:
![image](https://user-images.githubusercontent.com/7207207/191426462-db81b3dd-cf91-4812-a5ce-d6b7bbbb9ce4.png)

Also, I did something cool (I don't know what) but now it suggests to lookup my measure in DAX.GUIDE.  I'm honored, but not that good in writing measures... :-)
![image](https://user-images.githubusercontent.com/7207207/191426783-cbf90a37-a19d-4548-93b0-9d878160513f.png)

**Comment by dgosbell** [comment]
Yes there is a setting in the options called "Intellisense Width %" which you can increase from 100% all the way upto 300%. But I'm also going to be increasing the base window size in the next release
